cfgov/unprocessed/css/organisms/read-more.less

```diff
@@ -43,19 +43,19 @@
 */
 
 .o-expandable__read-more-mobile {
-    .o-expandable_target__collapsed {
+    .o-expandable_target__collapsed,
+    .o-expandable_content__collapsed:after {
         display: none;
     }
 
-    .o-expandable_content__collapsed {
+    .o-expandable_content {
+        // These two !important values override basic expandable styling,
+        // because these don't function like expandables on sm+ screens.
+        display: block !important;
         max-height: 100% !important;
```

Comment:
Is it an issue that this wasn't moved, like the `:after` was?

Reply:
Seems to work fine.

Reply:
Yeah, I just tried it and tested the functionality and it worked /shrug